### PR TITLE
【BAAI】add std llava training case

### PR DESCRIPTION
**Pull request by nrikoh**
initial version

### CODE REVIEW

File: training/benchmarks/llava1.5_13b/deepspeed/model/apply_delta.py
Comment (shh2000) [inline training/benchmarks/llava1.5_13b/deepspeed/model/apply_delta.py:1]: any changes compared with the origin file from llava1.5 github repo? if not, we can "git clone" in env.sh, then simply move it, use it, no need to copy it into flagperf.

Comment (shh2000) [reply training/benchmarks/llava1.5_13b/deepspeed/model/apply_delta.py:1]: the same to other files under model/

File: training/benchmarks/llava1.5_13b/deepspeed/configA800.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Common arguments
+theoryflops = 312000000000000.0
+model_name_or_path = "/root/llava1.5_pretrain/data/LLaVA-Pretrain/checkpoints/vicuna-13b-v1.5"
+vision_tower = "/root/llava1.5_pretrain/data/LLaVA-Pretrain/openai/clip-vit-large-patch14-336"
```
Comment (shh2000) [inline training/benchmarks/llava1.5_13b/deepspeed/configA800.py:4]: all path should be relative path to "data_dir". user can only set PERF_PATH and DATA_DIR writable, while other directories remain read-only

**Comment by shh2000** [comment]
in benchmarks/case/framework/README.md, LLaVA's origin license(apache 2.0) should be mentioned and placed. if any question, you can refer to resnet50

File: training/benchmarks/llava1.5_13b/deepspeed/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+transformers==4.37.2
+accelerate==0.26.1
+deepspeed==0.13.1
+flash-attn==2.1.0
+SentencePiece
```
Comment (shh2000) [inline training/benchmarks/llava1.5_13b/deepspeed/requirements.txt:5]: need version

File: training/nvidia/llava1.5_13b-deepspeed/README.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+| 硬件设备简称 | nvidia A800                |                                    |
+| 硬件存储使用 | mem,见“性能指标”           | 通常称为“显存”,单位为GiB           |
+| 计算使用率 | MFU,见“性能指标”           | 参见PaLM论文定义 |
+| **吞吐量**   | **token/p/s,见“性能指标”** | 平均单卡每秒处理的token数          |
+| MMMU（val）结果           | acc(推理/验证)   | MMMU（val）回答准确率                   |
+* 性能指标
+
+| 配置                |  fix_hp           | token/p/s | loss | mem       |acc(MMMU) |MFU       |
+| ------------------- | ---------------- | ------ | ------- | --------- | --------- |--------- |
+| A800单机8卡（1x8）（pretrain）  |  /  | 2270.33 | 2.0083 | 76/81 | / |18.91%|
```
Comment (upvenly) [inline training/nvidia/llava1.5_13b-deepspeed/README.md:57]: 不做finetinue的结果不行是吧

Comment (shh2000) [reply training/nvidia/llava1.5_13b-deepspeed/README.md:57]: 因此无finetune结果接近初始随机结果，无法用于判定功能正确性

**Pull request merged**